static/css/components/work.less

```diff
@@ -47,7 +48,7 @@ div.editionAbout {
 }
 
 .work-subtitle{
-  margin-top:0px;
+  margin-top:0;
```

Comment:
I was surprised stylelint didnt complain about the space here. Is there a rule we could enable to catch this issue and have it fixed?